kompose.spec

```diff
@@ -23,12 +12,15 @@
 %global with_unit_test 0
 %endif
```

Comment:
so what I am proposing to do as part of this rpm build is to build kompose with bundled deps under the vendor folder (as they do upstream). Which one of these macros do I need to set?

I notice openshift origin package builds from bundled deps and does not have a devel package. Is there a reason for that?